src/main/java/com/lotto/util/LottoUtil.java

```diff
@@ -0,0 +1,30 @@
+package com.lotto.util;
+
+import java.util.HashSet;
+import java.util.Random;
+import java.util.Set;
+
+public class LottoUtil {
+    private static final int NUMBER_OF_LOTTO = 6;
+    private static final Random random = new Random();
```

Comment:
상수 객체 이므로 대문자로 표기 되면 좋을 것 같습니다 😄 
```suggestion
    private static final Random RANDOM = new Random();
```